resources/js/Shared/Components/FavoriteButton.vue

```diff
@@ -42,7 +42,7 @@ const toggleFavorite = async () => {
     result.value = !result.value
 
     if (result.value === false) {
-      toast.success(__('City removed from favorites.'))
+      toast.error(__('City removed from favorites.'))
```

Comment:
Why as error? Maybe better `info` or `success`.

Reply:
I changed this error to info